src/screens/Onboarding/OnboardingItem.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  useWindowDimensions,
+  ImageSourcePropType,
+} from "react-native";
+import { COLORS, FONTS, SIZES } from "../../styles/theme";
+
+interface Props {
+  id: string;
+  title: string;
+  description: string;
+  image: ImageSourcePropType;
+}
+
+const OnboardingItem: React.FC<Props> = ({ id, title, description, image }) => {
+  const { width } = useWindowDimensions();
+  return (
+    <View style={(styles.container, { width })}>
+      <Image
+        source={image}
+        style={[styles.image, { width, resizeMode: "contain" }]}
+      />
+      <View style={{ flex: 0.3 }}>
+        <Text style={styles.title}>{title}</Text>
+        <Text style={styles.description}>{description}</Text>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
```

Comment:
@AlphonceMutebi @Allen-Muhani @Chegemartinx 
The background color should not be specified here.
Should be in a central file.

Also the fontsize (line 47), fontweight (line 48), and so on, should not be defined here.